packages/terra-header/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2018 - 2019 Cerner Innovation, Inc.
```

Comment:
This should be 2019

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2545/commits/b4c876c6792a868e1528f1d9d08f59de74dff3af#diff-5dd7267d45043f905616b10c76d917eeL1